pkg/portal/graphql/app.go

```diff
@@ -298,6 +299,12 @@ var nodeApp = node(
 
 					stripeError, err := ctx.StripeService.GetLastPaymentError(*customerID)
 					if err != nil {
+						if errors.Is(err, libstripe.ErrNoSubscription) {
```

Comment:
I think GetLastPaymentError should return (nil, nil) for this case. So this checking is not needed here.